src/Strategy/Swift/PackageSwift.hs

```diff
@@ -197,20 +199,45 @@ parsePackageSwiftFile = do
 
 -- | Analysis
 -- *
-analyzePackageSwift :: (Has ReadFS sig m, Has Diagnostics sig m) => Path Abs File -> m (Graphing.Graphing Dependency)
-analyzePackageSwift manifestFile =
-  do
-    manifestContent <-
-      context
-        "Identifying dependencies in Package.swift"
-        $ readContentsParser parsePackageSwiftFile manifestFile
-    context "Building dependency graph" $
-      pure $ buildGraph manifestContent
+analyzePackageSwift :: (Has ReadFS sig m, Has Diagnostics sig m) => Path Abs File -> Maybe (Path Abs File) -> m (Graphing.Graphing Dependency)
+analyzePackageSwift manifestFile resolvedFile = do
+  manifestContent <-
+    context
+      "Identifying dependencies in Package.swift"
+      $ readContentsParser parsePackageSwiftFile manifestFile
+
+  packageResolvedContent <- case resolvedFile of
+    Nothing -> pure Nothing
+    Just packageResolved ->
+      do
+        context "Identifying dependencies in Package.swift" $
+          readContentsJson packageResolved
+
+  context "Building dependency graph" $
+    pure $ buildGraph manifestContent packageResolvedContent
```

Comment:
[nit]: Some of the formatting here is really strange.  Can you collapse these into one-liners and then let the formatter break it apart?  If that doesn't change it, then it should be fine.

Reply:
Yeah - it reads awk. I hit `make fmt` per commit so, this is what formatter is resolving to. Will try to make it more readable.